plugins/carlabase/carla.cpp

```diff
@@ -501,8 +500,20 @@ CarlaInstrumentView::~CarlaInstrumentView()
 
 void CarlaInstrumentView::toggleUI(bool visible)
 {
-    if (fHandle != NULL && fDescriptor->ui_show != NULL)
+    if (fHandle != NULL && fDescriptor->ui_show != NULL) {
+// TODO: remove when fixed upstream
+// change working path to location of carla.dll to avoid conflict with lmms
+#if defined(CARLA_OS_WIN32) || defined(CARLA_OS_WIN64)
+        if (visible) {
+            QString backupDir = QDir::currentPath();
+            QDir::setCurrent(QFileInfo(carla_get_library_filename()).dir().absolutePath());
```

Comment:
FYI you can use `carla_get_library_folder` here

Reply:
Thanks, done.

Reply:
Hmm... maybe not, since f207715, it's no longer working.  Hmm...

Reply:
Fixed 2f7b608 :)